src/Foundation.Cms/Blocks/GoogleMapsEmbedBlock.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using EPiServer.Core;
+using EPiServer.DataAbstraction;
+using EPiServer.DataAnnotations;
+
+namespace Foundation.Cms.Blocks
+{
+    [ContentType(DisplayName = "Google Maps Embed Block",
+        GUID = "8fc31051-6d22-4445-b92d-7c394267fa49",
+        Description = "Display Google Maps Embed",
+        GroupName = CmsGroupNames.SocialMedia)]
+    public class GoogleMapsEmbedBlock : FoundationBlockData
+    {
+        [Display(Name = "API Key")]
+        public virtual string ApiKey { get; set; }
+
+        [Display(Name = "Search term")]
+        public virtual string SearchTerm { get; set; }
+
+        public virtual double Width { get; set; }
```

Comment:
We can set width 100% so we can control the Block using display mode, we just care about the height.